### DISCUSSION TITLE
recommended-posts-for-jekyll/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#333] [General]
# recommended-posts-for-jekyll/

想为本博客添加相关文章功能，因为本站是通过Jekyll搭建的静态站，所以首先想到通过Jekyll相关插件解决，没想到搜遍全网居然未找到可用的插件，无奈最后手敲实现，以下记录折腾过程。

https://www.coderli.com/recommended-posts-for-jekyll/

**Comment by lihongzheshuai** [comment]
更新主题
